test/benchmark/reasoner/iam/complex/ConjunctionStructureTest.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (C) 2022 Vaticle
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+package com.vaticle.typedb.core.reasoner.benchmark.iam.complex;
+
+import com.vaticle.typedb.core.reasoner.benchmark.iam.common.Benchmark;
+import com.vaticle.typedb.core.reasoner.benchmark.iam.common.BenchmarkRunner;
+import com.vaticle.typedb.core.reasoner.benchmark.iam.common.QueryParams;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class ConjunctionStructureTest {
+    static final Path RESOURCE_DIRECTORY =  Paths.get("test", "benchmark", "reasoner", "iam", "complex");
+    private static final Path COMMON_RESOURCE_DIR = Paths.get("test", "benchmark", "reasoner", "iam", "resources");
+
+    private static final String database = "iam-benchmark-conjunctions";
+    private static final BenchmarkRunner benchmarker = new BenchmarkRunner(database);
```

Comment:
I think a better name would be `runner` or `benchmarkRunner`, we don't really have the convention of verbalising nounds with '-er' suffixes